src/web/app/Page.re

```diff
@@ -196,6 +230,59 @@ module Update = {
           editors,
         };
       };
+    //TODO: Figure out how below is impacted by new global undo/redo
```

Comment:
the below changes were on code that's no longer extant due to matt's new global undo/redo system... maybe ask him if any of the assistant logic beneath is necessary?